packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -318,7 +313,6 @@ def test_dcos_diagnostics_report(dcos_api_session):
 
 
 @pytest.mark.parametrize('use_legacy_api', [False, True])
-@pytest.mark.xfail("config.getoption('--windows-only')", strict=True, reason="D2IQ-64753")
 def test_dcos_diagnostics_bundle_create_download_delete(dcos_api_session, use_legacy_api):
```

Comment:
This one fails because of
```
>               raise AssertionError('summaryErrorsReport.txt must be empty. Got {}'.format(log_data))
E               AssertionError: summaryErrorsReport.txt must be empty. Got could not collect adminrouter: could not read adminrouter logs from journal: there is no journal on Windows
E               could not collect dcos-diagnostics: could not read dcos-diagnostics logs from journal: there is no journal on Windows
E               could not collect docker: could not read docker logs from journal: there is no journal on Windows
E               could not collect mesos-agent: could not read mesos-agent logs from journal: there is no journal on Windows
E               could not collect telegraf: could not read telegraf logs from journal: there is no journal on Windows
E               could not collect WinRM: could not read WinRM logs from journal: there is no journal on Windows
E               could not collect 5051-overlay-agent_overlay.json: unable to fetch http://EC2AMAZ-6A2OQAU:5051/overlay-agent/overlay. Return code 404. Body: 404 Not Found.
```
How are we collecting logs on Windows, @DominikDary?

Reply:
with dcos-diagnostics, config you find for open and windows here: 
https://github.com/dcos/dcos/blob/master/gen/dcos-config-windows.yaml#L65-L96

Reply:
I'm a little uncertain about the expected behaviour.